### DISCUSSION TITLE
Image naming scheme

### DISCUSSION THREAD

**Discussion by maniator** [maniator/dind-buildx#9] [General]
Should we rethink how the images are named?

Now that docker is configurable along with the version of buildx (of of #5 )? (As I have in my PR for #8 we probably don't need alpine)

Idk a good scheme to name them

Any ideas?
Just naming after the buildx version is probably not the best approach.

**Comment by joakimr-axis** [comment]
I agree, but cannot say if approach A, B or C would be the best one; isn't that always the case with naming and labeling software, no matter how you choose, it will turn out to have flaws in some cases.

**Reply by maniator** [reply]
Ha true
